app/views/layouts/application.html.erb

```diff
@@ -12,27 +12,27 @@
     <div class="content">
       <% if user_signed_in? %>
         <%= link_to t(:global_proposition_header), main_app.root_path, id: "proposition-name" %>
-        <a role="button" href="#proposition-links" class="js-header-toggle menu" aria-controls="navigation" aria-label="Show or hide Top Level Navigation">Menu</a>
-          <nav id="proposition-menu">
-            <ul id="proposition-links" aria-label="Top Level Navigation">
+        <a role="button" href="#proposition-links" class="js-header-toggle menu" aria-controls="proposition-menu" aria-label="Show or hide Top Level Navigation">Menu</a>
```

Comment:
W3C error here was about the `aria-controls` pointing to a destination that does not exist